crates/codegen/runtime/cargo/src/runtime/bindings/mod.rs

```diff
@@ -1,10 +1,193 @@
 #[path = "generated/binding_rules.rs"]
 mod binding_rules;
 
-use metaslang_graph_builder::ast;
-pub use metaslang_graph_builder::functions::Functions;
-pub use metaslang_graph_builder::{ExecutionConfig, ExecutionError, NoCancellation, Variables};
+pub mod graph_builder {
+    use metaslang_graph_builder::{ast, functions, graph};
+
+    use crate::cst::KindTypes;
+
+    pub type File = ast::File<KindTypes>;
+    pub type Functions = functions::Functions<KindTypes>;
+    pub type Graph = graph::Graph<KindTypes>;
+
+    pub use metaslang_graph_builder::{ExecutionConfig, NoCancellation, Variables};
+}
+
+use std::collections::{BTreeSet, HashMap};
```

Comment:
I wonder if there is anything in `Bindings` that is Solidity-specific?
If not, what do you think about moving it to the `graph_builder` crate? this way, we don't need to add `stack_graphs` crate to `slang_solidity` or `slang_testlang`.

Reply:
I don't think there is. I thought about moving the `Bindings` struct into the `metaslang_graph_builder` crate, but I don't feel it belongs there. Unless we want to rename the crate to `metaslang_bindings`, and restructure it. Already putting the `stack_graph` module there feels a bit out of place.

Reply:
I moved the bindings code into the new `metaslang_bindings` crate. All that is left in the runtime is the construction function because we need to provide the bindings rules source somehow.